objgraph.py

```diff
@@ -775,7 +794,7 @@ def _short_typename(obj):
 
 
 def _long_typename(obj):
-    objtype =_get_obj_type(obj)
+    objtype = _get_obj_type(obj)
```

Comment:
I missed this when I merged the PR that introduced `_get_obj_type()`!  That's an excellent example of why adding a style checker to the Travis test suite is a good thing.